src/customer/create-customer-strategy-registry.ts

```diff
@@ -9,35 +9,37 @@ import { PaymentMethodActionCreator, PaymentMethodRequestSender } from '../payme
 import { AmazonPayScriptLoader } from '../payment/strategies/amazon-pay';
 import { createBraintreeVisaCheckoutPaymentProcessor, VisaCheckoutScriptLoader } from '../payment/strategies/braintree';
 import { ChasePayScriptLoader } from '../payment/strategies/chasepay';
+import createGooglePayPaymentProcessor from '../payment/strategies/googlepay/create-googlepay-payment-processor';
 import { MasterpassScriptLoader } from '../payment/strategies/masterpass';
 import { RemoteCheckoutActionCreator, RemoteCheckoutRequestSender } from '../remote-checkout';
 
-import CustomerActionCreator from './customer-action-creator';
-import CustomerRequestSender from './customer-request-sender';
-import CustomerStrategyActionCreator from './customer-strategy-action-creator';
+import { CustomerActionCreator, CustomerRequestSender, CustomerStrategyActionCreator } from './';
 import {
     AmazonPayCustomerStrategy,
     BraintreeVisaCheckoutCustomerStrategy,
     ChasePayCustomerStrategy,
     CustomerStrategy,
     DefaultCustomerStrategy,
+    GooglePayBraintreeCustomerStrategy,
     MasterpassCustomerStrategy,
+    SquareCustomerStrategy
```

Comment:
Did this one sneak in? What is the Square addition tied to?

Reply:
We need it to make the sign out link works (the one that says **Sign out from Square to see other payment methods**) on shipping step